### PR TITLE
Bump mdbook

### PR DESCRIPTION
**Pull request by lzutao**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-book
-.blacksmith-cache.json
```
Comment (XAMPPRocky) [inline .gitignore]: Why would `.blacksmith-cache.json` be removed? Don't we still generate this file?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 language: rust
 dist: xenial
 
+branches:
+  only:
+    - master
```
Comment (XAMPPRocky) [inline .travis.yml]: I'm not sure this is what we want? Won't this disable checks for PRs that aren't on a master branch?

File: .gitignore (hunk 1 above)
Comment (lzutao) [reply .gitignore]: Guess it ges loss on rebase

File: .travis.yml (hunk 2 above)
Comment (lzutao) [reply .travis.yml]: I just move it around, not add it.

File: .gitignore (hunk 1 above)
Comment (lzutao) [reply .gitignore]: Done.

File: .travis.yml (hunk 2 above)
Comment (XAMPPRocky) [reply .travis.yml]: Well then would you be able to remove it from the changes?

Comment (lzutao) [reply .travis.yml]: Yeah, I can. But I find putting `branches` before `script` is more accessible and readable.

Comment (XAMPPRocky) [reply .travis.yml]: Ok, seems fine to me.

**Comment by XAMPPRocky** [comment]
Thank you again for your PR!

**Pull request merged**